enumerables.rb

```diff
@@ -0,0 +1,97 @@
+module Enumerable
+  def my_each()
+    i = -1
```

Comment:
* [ ] `my_each`  should be identical and returns the same thing as ruby's [`each`](https://ruby-doc.org/core-2.6.4/Array.html#method-i-each). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
1.  ```p [1,2,3,4,5].my_each{|a| a}```  should return same results as  ``` p [1,2,3,4,5].each{|a| a} ```
2. ```p [1,2,3,4,5].my_each ```  should return same results as  ``` p [1,2,3,4,5].each ```